### ISSUE TITLE
Video: Support rendering out a single frame, as JPEG/PNG

### ISSUE THREAD

**Issue by jonnor** [imgflo/imgflo-server#109]
Should seek to a specified frame. Probably might want to make that `video_start_frame` input. And probably should work when outputting video also. At which point probably should also add `video_end_frame` for completeness...

**Comment by bergie** [comment]
Sounds like slightly overlapping functionality with https://github.com/the-grid/iconographer ?

**Comment by bergie** [comment]
Another interesting thing would be to request a set of frames and get an animated gif containing those, kind of like how Google+ does video previews

**Comment by jonnor** [comment]
GIFs should generally die... For that usecase, extract frames as JPEG and it play forward with JS.

Or could output as mp4/webm, with a very low framerate. Problem is, you want to be able to control the timing in the UI, and not have to re-render everything if you change your mind about it.

**Comment by jonnor** [comment]
@bergie this does zero smarts, just obeys client request. iconographer is for the smarts.

**Comment by bergie** [comment]
Fair point. So then implementing this feature (extracting given frame as jpeg) would already do the trick

**Comment by jonnor** [comment]
Yes, assuming the client knows how many frames long the video is :)
